comp/core/status/statusimpl/status_test.go

```diff
@@ -927,7 +927,7 @@ Status render errors
 
 	for _, testCase := range testCases {
 		t.Run(testCase.name, func(t *testing.T) {
-			bytesResult, err := statusComponent.GetStatusBySection(testCase.section, testCase.format, false)
+			bytesResult, err := statusComponent.GetStatusBySection([]string{testCase.section}, testCase.format, false)
```

Comment:
Could we add a test case that uses multiple sections?
Another possible test is to make sure if the section do not exists or is empty dopes not cause any issue.

Reply:
Done in last commit